spec/requests/contributors_spec.rb

```diff
@@ -93,7 +93,7 @@
     end
 
     context 'given a manually created contributor' do
-      let(:contributor) { create(:contributor, :manually_created) }
+      let(:contributor) { build(:contributor, data_processing_consent: false).tap { |c| c.save(validate: false) } }
```

Comment:
This is to workaround the consent validation on `create`, right?

Reply:
Si